gtsam/hybrid/HybridBayesTree.cpp

```diff
@@ -24,6 +24,7 @@
 #include <gtsam/hybrid/HybridBayesTree.h>
 #include <gtsam/inference/BayesTree-inst.h>
 #include <gtsam/inference/BayesTreeCliqueBase-inst.h>
```

Comment:
move inst to bottom

Reply:
I tried but the formatter keeps moving it back 😕